### PR TITLE
Conflict resolution  master => Feature/842 ngsi ld

### PR DESCRIPTION
**Pull request by jason-fox**
Solving merge conflicts `master` => `nsgi-ld`

### CODE REVIEW

File: doc/api.md [hunk 1]
```diff
@@ -203,25 +205,27 @@ Note that there is a 1:1 correspondence between payload fields and DB fields (bu
...
+| `entity_name`             | `name`               | Name of the entity representing the device in the Context Broker                                                                                                                 | ParkLamplight12                               |
+| `entity_type`             | `type`               | Type of the entity in the Context Broker                                                                                                                                         | Lamplights                                    |
+| `timezone`                | `timezone`           | Time zone of the sensor if it has any                                                                                                                                            | America/Santiago                              |
+| `timestamp`               | `timestamp`          | Optional flag about whether or not to addthe TimeInstant attribute to the device entity created, as well as a TimeInstant metadata to each attribute, with the current timestamp | true                                          |
```
Comment (fgalan) [inline doc/api.md:216]: Diff shows:

imagen

Is the removal of the sentence about NGSI-LD ("With NGSI-LD ...") correct?

File: doc/installationguide.md [hunk 2]
```diff
@@ -227,8 +226,8 @@ used for the same purpose. For instance:
     any unexpected error.
 -   **singleConfigurationMode**: enables the Single Configuration mode for backwards compatibility (see description in
     the Overview). Default to false.
--   **timestamp**: if this flag is activated:
-    -   For NGSIv1/NGSIv2, the IoT Agent will add a `TimeInstant` metadata attribute to all the attributes updated from device information. This flag is overwritten by `timestamp` flag in group or device
+-   **timestamp**: if this flag is activated, the IoT Agent will add a 'TimeInstant' metadata attribute to all the
+    attributes updated from device information. This flag is overwritten by `timestamp` flag in group or device
```
Comment (fgalan) [inline doc/installationguide.md:230]: Diff shows:

imagen

Is correct with the current wording? I think two bullets (one for NGSIv1/NGSIv2 and other for NGSI-LD) looks better.

File: doc/api.md (hunk 1 above)
Comment (jason-fox) [reply doc/api.md]: Fixed  ddb8cfe

File: doc/installationguide.md (hunk 2 above)
Comment (jason-fox) [reply doc/installationguide.md]: Fixed 21a657a

**Pull request merged**